### ISSUE TITLE
[Bug]Crash after removing the Ublock Add-on

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8119]
## Steps to reproduce
1.  Install the uBlock Origin Add-on
2. Remove the uBlock Origin Add-on 

### Expected behavior
The add-on is properly removed.

### Actual behavior
Crash

### Device information

* Android device: OnePlus A3 (Android 6.0.1)
* Fenix version: Nightly #20360606 from 2/5

### Notes
❗ **Reproducible only after a couple of attempts.**

► Video
![InShot-20200205-174315115](https://user-images.githubusercontent.com/51314259/73857576-d0b6c580-483f-11ea-93fc-bd8d56178c1f.gif)

►Log
```kotlin
--------- beginning of crash
02-05 17:39:53.062 22344-22344/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.nightly, PID: 22344
    org.mozilla.geckoview.GeckoResult$UncaughtException: java.lang.IllegalStateException: Fragment InstalledAddonDetailsFragment{352b931} (259f56b0-a781-46f2-9f25-a121ad2c6e10)} not attached to a context.
        at org.mozilla.geckoview.GeckoResult.dispatchLocked(GeckoResult.java:4)
        at org.mozilla.geckoview.GeckoResult.completeExceptionally(GeckoResult.java:4)
        at org.mozilla.geckoview.GeckoResult.lambda$thenInternal$2$GeckoResult(GeckoResult.java:10)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$gwCgOUK_EYQn2g6GolfZvo6A_WE.run(lambda)
        at org.mozilla.geckoview.GeckoResult.lambda$dispatchLocked$3(GeckoResult.java:2)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$PehjG2jgyDa_p37vZrKrSi2I94s.run(lambda)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:152)
        at android.app.ActivityThread.main(ActivityThread.java:5507)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.lang.IllegalStateException: Fragment InstalledAddonDetailsFragment{352b931} (259f56b0-a781-46f2-9f25-a121ad2c6e10)} not attached to a context.
        at androidx.fragment.app.Fragment.requireContext(Fragment.java:2)
        at androidx.fragment.app.Fragment.getResources(Fragment.java:1)
        at androidx.fragment.app.Fragment.getString(Fragment.java:2)
        at -$$LambdaGroup$ks$NW11xwGQjlJ_TLZipM3nRB7Y0SU.invoke(com.android.tools.r8.jetbrains.kotlin-style lambda group:229)
        at -$$LambdaGroup$ks$em8DxEANWiIxVC5T2xL3UgsR3jA.invoke(com.android.tools.r8.jetbrains.kotlin-style lambda group:36)
        at -$$LambdaGroup$js$4TkCmOpDQ4Op5MPl4rXDRUWEFCs.onValue(java-style lambda group:12)
        at org.mozilla.geckoview.GeckoResult.lambda$thenInternal$2$GeckoResult(GeckoResult.java:2)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$gwCgOUK_EYQn2g6GolfZvo6A_WE.run(lambda) 
        at org.mozilla.geckoview.GeckoResult.lambda$dispatchLocked$3(GeckoResult.java:2) 
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$PehjG2jgyDa_p37vZrKrSi2I94s.run(lambda) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:152) 
        at android.app.ActivityThread.main(ActivityThread.java:5507) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
02-05 17:39:53.062 22344-22344/? D/AppTracker: App Event: crash
02-05 17:39:53.079 1270-1312/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@6ba03ba
02-05 17:39:53.079 1270-1312/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@632726b
02-05 17:39:53.087 1270-1312/? W/ActivityManager:  Dismiss app error dialog : org.mozilla.fenix.nightly
02-05 17:39:53.087 1270-1312/? W/AppOps: Finishing op nesting under-run: uid 1000 pkg android code 24 time=0 duration=0 nesting=0
02-05 17:39:53.090 22344-22344/? I/Process: Sending signal. PID: 22344 SIG: 9
02-05 17:39:53.109 1270-22645/? D/DropBoxManagerService: file :: /data/system/dropbox/data_app_crash@2020-02-05-17_39_53_071.txt
02-05 17:39:53.160 1270-2216/? D/GraphicsStats: Buffer count: 5
02-05 17:39:53.160 1270-1822/? I/ActivityManager: Process org.mozilla.fenix.nightly (pid 22344) has died
02-05 17:39:53.162 1270-2450/? I/WindowState: WIN DEATH: Window{9c3368a u0 org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App}
02-05 17:39:53.164 22412-22412/? I/ServiceChildProcess: Service has been unbound. Stopping.
02-05 17:39:53.176 1270-1822/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@61cc1e0
02-05 17:39:53.177 1270-1822/? I/ActivityManager: Start proc 22646:org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashReporter/u0a208 for activity org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity```

**Comment by csadilek** [comment]
This has the same cause as https://github.com/mozilla-mobile/fenix/issues/8097. Fix incoming...

**Comment by Amejia481** [comment]
@AndiAJ this should be fixed in master now, could you please take a look?

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20370604 from 2/6`** using a OnePlus A3 (Android 6.0.1)

After approx. 25 installs and uninstalls no crash encountered.
